docs/repositories-configure/file-extensions.md

```diff
@@ -6,6 +6,9 @@ description: If your repository has source files with unrecognized extensions, y
 
 If your repository has source files with unrecognized extensions, you can configure Codacy to include them in the next analysis:
 
+!!! note
```

Comment:
I believe we are excessively highlighting this detail. Consider moving this note to the bottom of the page. And we could add a link to **Semgrep** to the provider's page (https://semgrep.dev/).

Note: Why do we need the `skip-vale` class here? Shouldn't we add the tool name to the [accept.txt](https://github.com/codacy/docs/blob/master/.github/styles/Vocab/Codacy/accept.txt) file instead? 🤔